docs/reference/commandline/cli.md

```diff
@@ -160,6 +160,14 @@ property is not set, the client falls back to the default table
 format. For a list of supported formatting directives, see
 [**Formatting** section in the `docker stats` documentation](stats.md)
 
+The property `secretFormat` specifies the default format for `docker
+secret ls` output. When the `--format` flag is not provided with the
+`docker secret ls` command, Docker's client uses this property. If this
+property is not set, the client falls back to the default table
+format. For a list of supported formatting directives, see
+[**Formatting** section in the `docker secret ls` documentation](secret_ls.md)
+
```

Comment:
Oh there's a sample `config.json` configuration file; would be good to have an example `secretFormat` in there;

<img width="978" alt="screen shot 2017-03-13 at 18 07 45" src="https://cloud.githubusercontent.com/assets/1804568/23866136/2c300bb2-0818-11e7-8628-320000b5072f.png">